demo/cache/cache-controller.js

```diff
@@ -1,8 +1,9 @@
-app.controller('CacheCtrl', function($scope, $rootScope, $http, FDP, Caching, Log) {    
+app.controller('CacheCtrl', function($scope, $rootScope, $http, FDP, Caching, Log, GENERAL_CONFIG) {    
   $scope.isCachingStarted = false;
   $scope.fairDataPoints=[{name: "RD connect FDP", url:"http://localhost:8084/fairdatapoint/fdp"}];
   $scope.searchEngineUrl=null;
-  
+  $scope.endPointBaseUrl = GENERAL_CONFIG.END_POINT_BASE_URL;
```

Comment:
This way you generalize  HttpEndPoint, getting its caching server through an argument rather than from config file / variable structure that might be architecture dependent (the config format may change).

Reply:
But it makes sense to do this way if this variable is dynamic but in the current implementation the endpoint url is static

Reply:
BTW the "HttpEndpoint" service layer implementation is specific to "blazegraph" I don't think the functions in this service layer works if you provide different triplestore endpoint. BTW the current implementation barely works with "blazegraph" :-p

Reply:
:) I got that feeling. There are a couple of paths that look blazegraph specific.